### PR TITLE
Create Queue_array.java

### LINKED ISSUE #2299: Queue implementation using Array in Java
As a participant of GSSoC, I want to work on this. Please assign this task to me.

### PR DESCRIPTION
**Pull request by Amitsharma45**
Fixes #2299 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- java
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by Riyagoel2** [comment]
Hi , I want to work on this issue in Java. 
Please assign it to me @jainaman224

File: Queue_Using_Arrays/Queue_array.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
+// Java program to implement a queue using an array 
+class Queue { 
+	private static int front, rear, capacity;
```
Comment (jainaman224) [inline Queue_Using_Arrays/Queue_array.java]: Use 4 space indentation.

File: Queue_Using_Arrays/Queue_array.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	public static void main(String[] args) 
+	{ 
+		// Create a queue of capacity 4 
+		Queue q = new Queue(4); 
+
+		// print Queue elements 
+		q.queueDisplay(); 
+
+		// inserting elements in the queue 
+		q.queueEnqueue(20);
```
Comment (jainaman224) [inline Queue_Using_Arrays/Queue_array.java]: Take input from user.

File: Queue_Using_Arrays/Queue_array.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+		q.queueDequeue(); 
+		System.out.printf("\n\nafter two node deletion\n\n"); 
+
+		// print Queue elements 
+		q.queueDisplay(); 
+
+		// print front of the queue 
+		q.queueFront(); 
+	} 
+}
```
Comment (jainaman224) [inline Queue_Using_Arrays/Queue_array.java]: Add sample input and output.

**Comment by jainaman224** [comment]
@Riyagoel2 Please another issue.

**Comment by somya-kapoor** [comment]
@Amitsharma45 Are you working on it?

**Comment by Amitsharma45** [comment]
@jainaman224 @somya-kapoor  update comlplete please review pr

File: Queue_Using_Arrays/Queue_array.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
+// Java program to implement a queue using an array 
+import java.io.*;
+class QueueArr
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_array.java:3]: Leave a line before it

File: Queue_Using_Arrays/Queue_array.java [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+{
+    static int i,front,rear,item,max=5,ch;
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_array.java:5]: Add space after ,
Add space around the operator

Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_array.java:5]: Add space after ,
Add space around the operator

File: Queue_Using_Arrays/Queue_array.java [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    static int a[]=new int[5];
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_array.java:6]: Add space around the operator

File: Queue_Using_Arrays/Queue_array.java [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+    QueueArr()
+    {
+        front=-1;
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_array.java:9]: Add space around the operator

File: Queue_Using_Arrays/Queue_array.java [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+        rear=-1;
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_array.java:10]: Add space around the operator

File: Queue_Using_Arrays/Queue_array.java [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+    }
+    public static void main(String args[])throws IOException
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_array.java:12]: Leave a line before it

File: Queue_Using_Arrays/Queue_array.java [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+    {
+ 
+        while((boolean)true)
+        {
+            try
+            {
+                System.out.println("Select Option 1.insert 2.delete 3.display 4.Exit");
+                BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_array.java:20]: Add space around the operator
Do this in whole file

File: Queue_Using_Arrays/Queue_array.java [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+                    delete();
+                    break;
+                case 3:
+                    display();
+                    break;
+                }
+            }
+        }
+    }
+    static void insert()
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_array.java:44]: Leave a line before it

File: Queue_Using_Arrays/Queue_array.java [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+    static void display()
+    {
+        System.out.println("Elements in the Queue are:");
+        for(int i=front+1; i<=rear; i++)
+        {
+            System.out.println(a[i]);
+        }
+    }
+}
+// sample input and output
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_array.java:86]: Leave a line before it

**Comment by somya-kapoor** [comment]
The issue number #2299 is not assigned to you.

**Comment by somya-kapoor** [comment]
If I am wrong, please inform me.

**Pull request closed without merging**